src/main/java/org/springframework/data/couchbase/config/AbstractCouchbaseConfiguration.java

```diff
@@ -273,6 +287,31 @@ public CouchbaseMappingContext couchbaseMappingContext(CustomConversions customC
 		return mappingContext;
 	}
 
+	/**
+	 * Creates a {@link Module} ObjectMapper mapper = new ObjectMapper();
+	 * mapper.registerModule(couchbaseJsonValueModule()); <----- ClusterEnvironment env = ClusterEnvironment.builder()
+	 * .jsonSerializer(new JacksonJsonSerializer(mapper)) .build();
+	 *
+	 * @throws Exception on Bean construction failure.
+	 * @return
+	 */
+	@Bean
+	public com.couchbase.client.core.deps.com.fasterxml.jackson.databind.Module couchbaseJsonValueModule() {
```

Comment:
does this have to be its own bean?

Reply:
This doesn't need to exist at all. Removed.